iree/turbine/kernel/wave/expansion/expansion.py

```diff
@@ -401,7 +401,6 @@ def add_get_results(trace: CapturedTrace):
             get_result = get_custom(
                 GetResult(reduction.fx_node, 0).add_to_graph(reduction.graph)
             )
-            get_result.vector_shapes = reduction.init_args[0].vector_shapes
```

Comment:
nice! this is because we are now doing getResult before index analysis and expansion, S.T we can let index analysis set the vector shapes?

Reply:
yup that's right.